test/e2e/instrumentation/events.go

```diff
@@ -88,7 +88,7 @@ var _ = common.SIGDescribe("Events API", func() {
 		The event is patched with a new note, the check MUST have the update note.
 		The event is deleted and MUST NOT show up when listing all events.
 	*/
-	ginkgo.It("should ensure that an event can be fetched, patched, deleted, and listed", func() {
+	framework.ConformanceIt("should ensure that an event can be fetched, patched, deleted, and listed", func() {
```

Comment:
You also need to add this to test/conformance/testdata/conformance.yaml

Reply:
Done.